variables.tf

```diff
@@ -30,6 +30,17 @@ variable "lambda_s3_bucket" {
   description = "The name of the S3 bucket where the lambda build artifact is stored"
 }
 
+variable "logging_bucket" {
```

Comment:
Doesn't this need a default value to maintain backward compatibility?

`"${var.application_name}-${var.environment}-${var.file_uploads_bucket}-logs"`

Reply:
I restored the local `file_uploads_s3_bucket_logs` variable and added some conditional logic there to preserve the default.